### DISCUSSION TITLE
官方开发文档啥时候能添加中文版的啊？

### DISCUSSION THREAD

**Discussion by elarbor** [nodejs/help#4653] [Ideas]
希望添加中文版的官方开发文档

**Comment by avivkeller** [comment]


**Reply by bnb** [reply]
@RedYetiDev this is definitely not the correct answer, I'm not sure why you're saying that it's harder to maintain docs in multiple languages because "NodeJS is written in English". I appreciate your enthusiasm to answer people, but the reasoning provided here isn't currently and hasn't ever been relevant to internationalization in the project - especially not when there are pretty explicit and objective answers about how the project does something like translation.

**Reply by avivkeller** [reply]
My apologies! I didn't mean to make a mistake. I apologize for my misunderstanding, and I'll remove my answer.

**Comment by bnb** [comment]
hey @chergn!

If you're looking to help with internationalization / translation, that work lands under the Website Working Group. You can find their documentation [here](https://github.com/nodejs/nodejs.org/blob/main/TRANSLATION.md). I don't believe we currently publish any translated langauges, as the tooling has been being changed/updated, @ovflowd can probably provide more context than I can.

**Reply by avivkeller** [reply]
I'm sorry for my mistake! @bnb, your answer is the correct one, and I should not have pre-empively replied with false information.

**Reply by ovflowd** [reply]
Heyo 👋 yes, I'd recommend reading the documentation that @bnb cited above. We haven't published any language for the time being due to internationalisation issues with Crowdin, but we're hopeful that this gets fixed soon!
